### PR TITLE
@ #39 | should use IMAGE_TAG argument for flexible FROM base image

### PR DESCRIPTION
**Pull request by hieptranquoc**
@hoatle please help me check this PR

connect #39

### CODE REVIEW

File: httpie-jwt-auth/base-alpine/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM python:2.7-alpine
+ARG PYTHON_VERSION=2.7
```
Comment (hoatle) [inline httpie-jwt-auth/base-alpine/Dockerfile]: ARG PYTHON_VERSION=2.7-alpine

File: mongo-connector/base/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM python:2.7-alpine
+ARG PYTHON_VERSION=2.7
```
Comment (hoatle) [inline mongo-connector/base/Dockerfile]: ARG PYTHON_VERSION=2.7-alpine

File: python-coveralls/base/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,6 @@
-FROM python:2.7-alpine
+
+ARG PYTHON_VERSION=2.7
```
Comment (hoatle) [inline python-coveralls/base/Dockerfile]: ARG PYTHON_VERSION=2.7-alpine

File: httpie-jwt-auth/base-alpine/Dockerfile (hunk 1 above)
Comment (hieptranquoc) [reply httpie-jwt-auth/base-alpine/Dockerfile]: updated!

File: mongo-connector/base/Dockerfile (hunk 2 above)
Comment (hieptranquoc) [reply mongo-connector/base/Dockerfile]: updated!

File: python-coveralls/base/Dockerfile (hunk 3 above)
Comment (hieptranquoc) [reply python-coveralls/base/Dockerfile]: updated!

**Comment by hieptranquoc** [comment]
@hoatle i have updated, please help me check this PR again

**Pull request merged**